pfrl/agents/goal_conditioned_td3.py

```diff
@@ -213,10 +241,11 @@ def update_policy_with_goal(self, batch):
         batch_state = batch["state"]
         batch_goal = batch["goal"]
         action_distrib = self.policy(torch.cat([batch_state, batch_goal], -1))
-        onpolicy_actions = action_distrib.rsample()
+        onpolicy_actions_normalized = action_distrib.rsample()
+        onpolicy_actions = self.scale * onpolicy_actions_normalized
         entropy_term = 0
         if self.add_entropy:
-            log_prob = action_distrib.log_prob(onpolicy_actions)
+            log_prob = action_distrib.log_prob(onpolicy_actions_normalized)
```

Comment:
Just curious, doesn't it have to be `onpolicy_actions` so that the `log_prob` values will be large which then influences the size of the rewards

Reply:
address offline.